### PR TITLE
[RM Ch08] Orchestration support for 6G Use cases

### PR DESCRIPTION
**Pull request by joaoarodrigues**

### CODE REVIEW

File: doc/ref_model/chapters/chapter08.rst [hunk 1]
```diff
@@ -335,6 +335,12 @@ Infrastructure as well as provide:
   other Cloud Providers;
 - network orchestration solutions to manage the interconnectivity across its own and other Cloud Provider networks.
 
+The different orchestration solutions that can be found either in different techonlogy domains (e.g. Core, Edge, Radio), or different planes (Management and Control) for Service, Network and Resource layers shall consider the following attributes (not limited to the below ones):
+
+- Interoperability: Necessary integration among the orchestration components which would guarantee either the lifecycle management operations or related operation delegation to different orchestration components across the orchestration
+- Flexibiliy: There are use cases which can only be executed if the operations can run disaggregated, otherwise, extreme requirements brough y the use cases could be challenging to achieve taking in account technological constrains (e.g. use cases demanding less than 1 ms of latency)
```
Comment (walterkozlowski) [inline doc/ref_model/chapters/chapter08.rst:341]: typo "brough y the use"

**Review by walterkozlowski — CHANGES_REQUESTED** [review]
Typo needs to be fixed.
I would suggest moving this paragraph to the end of the subsection.  As it is, it breaks the flow.

File: doc/ref_model/chapters/chapter08.rst (hunk 1 above)
Comment (joaoarodrigues) [reply doc/ref_model/chapters/chapter08.rst]: It was now corrected in '[2f2e982](https://github.com/cntt-n/CNTT/pull/3254/commits/2f2e9826aabb599ea904427ae80f1557e19b4eeb)'

**Comment by joaoarodrigues** [comment]
> Typo needs to be fixed.
> I would suggest moving this paragraph to the end of the subsection. As it is, it breaks the flow.

Done [2467a8e](https://github.com/cntt-n/CNTT/pull/3254/commits/2467a8eb113f9ccea905e317e4452325aa18e894)

**Review by walterkozlowski — APPROVED** [review]

**Comment by walterkozlowski** [comment]
At RM Meeting pn 23 November 2022: discussion that we do not cover orchestration in RM; discussion to be continued post Nile

File: doc/ref_model/chapters/chapter08.rst [hunk 2]
```diff
@@ -350,6 +350,12 @@ criteria for inclusion of a standard/technology is defined. These standards/tech
 - not  be the sole proprietary property of a vendor/company
 - not be focused on "Transactions/Conversations" or "User/Data Plane" standards (typically IETF, IEEE,
   MEF/Carrier Ethernet etc.)
+  
+The different orchestration solutions that can be found either in different techonlogy domains (e.g. Core, Edge, Radio), or different planes (Management and Control) for Service, Network and Resource layers shall consider the following attributes (not limited to the below ones):
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter08.rst:354]: Please wrap the lines to 120 chars.

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_model/chapters/chapter08.rst (hunk 2 above)
Comment (joaoarodrigues) [inline doc/ref_model/chapters/chapter08.rst:354]:
```suggestion
The different orchestration solutions that can be found either in different techonlogy domains (e.g. Core, Edge, 
Radio), or different planes (Management and Control) for Service, Network and Resource layers shall consider 
the following attributes (not limited to the below ones):
```

**Review by CsatariGergely — APPROVED** [review]

**Review by petorre — APPROVED** [review]

File: doc/ref_model/chapters/chapter08.rst [hunk 3]
```diff
@@ -350,6 +350,14 @@ criteria for inclusion of a standard/technology is defined. These standards/tech
 - not  be the sole proprietary property of a vendor/company
 - not be focused on "Transactions/Conversations" or "User/Data Plane" standards (typically IETF, IEEE,
   MEF/Carrier Ethernet etc.)
+  
+The different orchestration solutions that can be found either in different techonlogy domains (e.g. Core, Edge, 
+Radio), or different planes (Management and Control) for Service, Network and Resource layers shall consider 
+the following attributes (not limited to the below ones):
+
+- Interoperability: Necessary integration among the orchestration components which would guarantee either the lifecycle management operations or related operation delegation to different orchestration components across the orchestration
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter08.rst:358]:
```suggestion
- Interoperability: among the orchestrators to support the lifecycle operations or the delegation of operations to other orchestrators
```

File: doc/ref_model/chapters/chapter08.rst [hunk 4]
```diff
@@ -350,6 +350,14 @@ criteria for inclusion of a standard/technology is defined. These standards/tech
...
+- Flexibiliy: There are use cases which can only be executed if the operations can run disaggregated, otherwise, extreme requirements brough by the use cases could be challenging to achieve taking in account technological constrains (e.g. use cases demanding less than 1 ms of latency)
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter08.rst:359]: this is a continuation from the text below the bullets
```suggestion
- Flexibility: to support uses that require disaggregated operations; such uses may involve technological constraints (e.g., less than 1 ms of latency)
```

File: doc/ref_model/chapters/chapter08.rst [hunk 5]
```diff
@@ -350,6 +350,14 @@ criteria for inclusion of a standard/technology is defined. These standards/tech
...
+- Intelligence: AI/ML support either for service lifecycle operation operation or service monitoring, otherwise, the capability of running an unlimited number of services (constrains can be brought by ecosystem/infrastructure variables like processing, memory, storage).
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter08.rst:360]: Not clear what this is trying to state. Can the approvers please step in and clarify?

File: doc/ref_model/chapters/chapter08.rst (hunk 4 above)
Comment (joaoarodrigues) [reply doc/ref_model/chapters/chapter08.rst:359]: > this is a continuation from the text below the bullets
Please check the following proposal

```suggestion
- Flexibiliy: to support uses that require disaggregated operations; such uses may involve technological constraints brought by the use cases (e.g. less than 1 ms of latency, bandwidth uplink rate at 1 Gbps, 5 nines availability ...)
```

**Pull request merged**